extensions/search-notion/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "search-notion",
+  "title": "Search Notion",
```

Comment:
Given the nature of this extension, I think we should mention in the title that this is without requiring admin access. This will prevent confusion when there is another Notion search that uses the official API, which requires admin access.